pkg/build/dockerfile-generator_test.go

```diff
@@ -214,7 +298,7 @@ func TestPorter_prepareDockerFilesystem(t *testing.T) {
 	require.NoError(t, err, "could not load manifest")
 
 	mp := mixin.NewTestMixinProvider()
-	g := NewDockerfileGenerator(c.Config, m, tmpl, mp)
+	g := NewDockerfileGenerator(c.Config, m, filepath.Join(c.Context.Getwd(), config.Name), tmpl, mp)
```

Comment:
Does NewDockerfileGenerator assume that the manifest path is absolute? Does it call filepath.Abs to ensure that it is? If we did that, one it would catch errors if people call it wrong and two would let us be lazy in our tests.